### DISCUSSION TITLE
1.2.6

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#34] [Announcements]
1. 优化 文件管理器UI , 增加文件管理器
2. 优化 下载中心UI, 增加 下载中心 开关、下载路径 等功能配置
3. 消息日志埋点处理，对 routergo 的 定时任务、重启、关机 等动作进行消息记录，主页的右上角消息可查看
4. 对移动端UI的优化
5. 优化内置登录页面和 iStoreNas 登录页面一致
6. Docker 增加任务列表，仅用于查看 routergo 操作docker的任务列表，大多数时间没用

**Comment by Fengerxuanxiao** [comment]
3、可以正常通知😘

**Reply by Fengerxuanxiao** [reply]
不过好像没办法手动清楚消息哦
